test/sandbox-test.js

```diff
@@ -167,6 +169,7 @@ describe("sinonSandbox", function () {
 
         // eslint-disable-next-line mocha/no-identical-title
```

Comment:
Why do they have identical titles? That's weird. I realize it isn't your change though.